olm-catalog/serverless-operator/manifests/operator_v1beta1_knativeserving_crd.yaml

```diff
@@ -2352,5 +2369,5 @@ spec:
       clientConfig:
         service:
           name: operator-webhook
-          namespace: default
+          namespace: knative-operator
```

Comment:
Is this change correct for us?

Reply:
Hm needs to be tested. IMHO we rewrite all namespaces to our own ones anyway. So probably yes?

Reply:
it's managed by OLM ? https://github.com/openshift-knative/serverless-operator/blob/473a9c576ba29bea28c653791e9541798e9d9dfc/olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml#L1283-L1304